airflow/providers/google/cloud/hooks/dlp.py

```diff
@@ -207,6 +206,9 @@ def create_dlp_job(
             until it is set to DONE.
         :type wait_until_finished: bool
         :rtype: google.cloud.dlp_v2.types.DlpJob
+        :param time_to_sleep_in_seconds: (Optional) Time to sleep between active checks
+            of the operation results. Defaults to 1.
```

Comment:
```suggestion
        :param time_to_sleep_in_seconds: (Optional) Time to sleep, in seconds, between active checks
            of the operation results. Defaults to 60.
```

Reply:
(yes, it's in the name already).

If we say "default to 60s" then someone may try to put `"5m" `, or another string value there if they aren't paying close attention

Reply:
Thanks, the new docstring will dispel the potential confusion.